packages/fakta/fakta-felles/src/components/personer/Personopplysninger.tsx

```diff
@@ -19,7 +19,7 @@ interface Props {
   harSammeAdresser?: boolean;
   navn?: string;
   sivilstand?: string;
-  dødsdato?: string;
+  dødsdato?: string | null;
```

Comment:
kan ein fjerna ? her?

Reply:
jepp

Reply:
nei, faktisk ikke, fordi dødsdato ikke skal vises i alle scenarioer hvor komponenten brukes. og noen steder har vi ikke dødsdato tilgjengelig, f.eks  for annenpart i medlemskapspanel